# (PART) Get RNA-seq data! {-}

# Data

`recount3` provides processed RNA-seq data for human and mouse in file formats similar to `recount2` (Figure \@ref(fig:recountWorkflowFig1)), which at its core is based on coverage bigWig files and exon-exon junction counts (see [Raw file](#raw-files) for more details). These two raw files power the whole `recount3` ecosystem. In `recount3`, we have provided coverage count files for several human and mouse annotations with samples grouped by study. Some large studies, like [GTEx](TODO) and [TCGA](TODO) have been fragmented at the tissue level to make the data more accessible.

```{r "recountWorkflowFig1", out.width="100%", fig.align="center", fig.cap = "Overview of the data available in recount2 and recount3. Reads (pink boxes) aligned to the reference genome can be used to compute a base-pair coverage curve and identify exon-exon junctions (split reads). Gene and exon count matrices are generated using annotation information providing the gene (green boxes) and exon (blue boxes) coordinates together with the base-level coverage curve. The reads spanning exon-exon junctions (jx) are used to compute a third count matrix that might include unannotated junctions (jx 3 and 4). Without using annotation information, expressed regions (orange box) can be determined from the base-level coverage curve to then construct data-driven count matrices.", echo = FALSE}
knitr::include_graphics("images/recountWorkflow_Figure1.png")
```

## Human

Annotations:

* Gencode v26
* Gencode v29
* RefSeq
* FANTOM6_cat
* ERCC
* SIRV

## Mouse
