# Raw Files

Explain the raw file formats we have and how they are organized

## Data source vs collection

A `data source` specifies where the data is hosted at, most commonly from the [Sequence Read Archive (SRA)](https://www.ncbi.nlm.nih.gov/sra). A `collection` is a manually curated set of samples from one or more studies. A collection has a custom metadata file where the curator(s) can specify metadata variables for the collection. In other words:

* `data_source`: samples from the original data origin
* `collection`: manually selected samples with curated collection-specific sample metadata

## Annotation files

Here are the direct links in case you are interested in downloading the annotation files directly.

```{r "ann_files", results='asis', echo = FALSE}
library("recount3")
library("purrr")

ann_files <- map_dfr(c("human", "mouse"), function(x) {
    map_dfr(c("gene", "exon"), function(y) {
        map_dfr(annotation_options(x), ~ 
                data.frame(
                    organism = x,
                    type = y,
                    annotation = .x,
                    file_extension = annotation_ext(
                        organism = x,
                        annotation = .x
                    ),
                    URL = locate_url_ann(
                        type = y,
                        organism = x,
                        annotation = .x
                    )
                )
        )
        
    })
})
knitr::kable(ann_files, row.names = FALSE, caption = "Annotation files")
```

On the R package, you can use `recount3::locate_url_ann()` to obtain these URLs.

The URL structure is:

`<recount3_url>`/`<organism>`/annotations/`<gene|exon>`_sums/`<organism>`.`<gene|exon>`_sums.`<annotation file extension>`.gtf.gz

These are the annotation file extensions; human:

* Gencode v26: `G026`
* Gencode v29: `G029`
* RefSeq: `ERCC`
* FANTOM6_cat: `F006`
* ERCC: `R109`
* SIRV: `SIRV`

Mouse:

* Gencode v23: `M023`

## Project-level count files

For every project, we have files at the gene, exon, and exon-exon junction expression levels. For genes and exons, we provide a file for each of the annotations. That is, for every project we provide:

* gene files: one count matrix per annotation
* exon files: one count matrix per annotation
* 3 exon-exon junction files: the sparse count matrix data in Matrix Market (MM) format, the small list of sample identifiers (IDs), and the exon-exon junctions coordinate information (RR file)

All these files can be located with `recount3::locate_url()`. The following R code creates a table with links to the files for the default annotation for each organism. Note that you can replace the annotation file extension (like `G026`) for the corresponding one for annotation annotation shown in the previous section (or use `recount3::annotation_ext()` to see available options).


```{r "project_raw_files", eval = FALSE}
## Obtain all available projects
projects <- rbind(
    recount3::available_projects("human"),
    recount3::available_projects("mouse")
)

## Locate the project raw files at the gene level using the default annotation
projects$gene <- apply(projects, 1, function(x)
    locate_url(
        project = x["project"],
        project_home = x["project_home"],
        type = "gene",
        organism = x["organism"],
        annotation = annotation_options(x["organism"])[1] # Use default annotation
    ))

## Locate the project raw files at the exon level using the default annotation
projects$exon <- apply(projects, 1, function(x)
    locate_url(
        project = x["project"],
        project_home = x["project_home"],
        type = "exon",
        organism = x["organism"],
        annotation = annotation_options(x["organism"])[1] # Use default annotation
    ))

## Locate the project raw exon-exon junction files
projects <-
    cbind(projects, do.call(rbind, apply(projects, 1, function(x) {
        x <-
            locate_url(
                project = x["project"],
                project_home = x["project_home"],
                type = "jxn",
                organism = x["organism"]
            )
        res <- data.frame(t(x))
        colnames(res) <-
            paste0("jxn_", gsub("^.*\\.", "", gsub("\\.gz", "", colnames(res))))
        return(res)
    })))
rownames(projects) <- NULL

## Dimensions of the table
dim(projects)
# [1] 18830    11

## Export
write.csv(projects, file = "recount3_raw_project_files_with_default_annotation.csv", row.names = FALSE)
```

As a teaser, here you can see the first 20 rows of this long table. Or you can download the CSV file to your computer from [GitHub](https://github.com/LieberInstitute/recount3-docs/blob/master/recount3_raw_project_files_with_default_annotation.csv).

```{r "project_raw_files_table", echo = FALSE, results = "asis"}
projects <- read.csv("recount3_raw_project_files_with_default_annotation.csv", row.names = NULL)
knitr::kable(head(projects, 20), row.names = FALSE, caption = "First 20 raw project files")
```

The URL structure is:

* gene: `<recount3_url>`/`<organism>`/data_sources/`<data_source>`/gene_sums/`<last 2 project letters or digits>`/`<project>`/`<data_source>`.gene_sums.`<project>`.`<annotation file extension>`.gz
* exon: `<recount3_url>`/`<organism>`/data_sources/`<data_source>`/exon_sums/`<last 2 project letters or digits>`/`<project>`/`<data_source>`.exon_sums.`<project>`.`<annotation file extension>`.gz
* junctions: `<recount3_url>`/`<organism>`/data_sources/`<data_source>`/junctions/`<last 2 project letters or digits>`/`<project>`/`<data_source>`.junctions.`<project>`.`<junction type: typically ALL>`.`<junction file extension: RR, MM or ID>`.gz ^[Only GTEx and TCGA have junction type `UNIQUE` available in addition to `ALL`.]

## Project-level metadata files

Every project from an original data source has 5 different sample metadata tables. These are:

* `project_meta` (`sra`, `gtex`, `tcga`): information mostly used by the R interface for locating files
* `recount_project`: information downloaded from the original data source, such as the SRA Run Table selector
* `recount_qc`: [quality check fields](#quality-check-fields) using the QC annotation
* `recount_seq_qc`: sequence quantily check fields
* `recount_pred`: curated and predicted sample information described in the recount3 manuscript

You can use the following R code to obtain the links to all these raw metadata files or use `recount3::locate_url()`.

```{r "metadata_files", eval = FALSE}
## Obtain all the metadata files
metadata_files <- do.call(rbind, apply(projects, 1, function(x) {
    x <-
        locate_url(
            project = x[["project"]],
            project_home = x[["project_home"]],
            type = "metadata",
            organism = x[["organism"]]
        )
    res <- data.frame(t(x))
    colnames(res) <-
        gsub("\\..*", "", gsub("^[a-z]+\\.", "", colnames(res)))
    
    colnames(res)[colnames(res) %in% unique(projects$file_source)] <-
        "project_meta"
    return(res)
}))
dim(metadata_files)
# [1] 18830     5

## Export
write.csv(metadata_files, file = "recount3_metadata_files.csv", row.names = FALSE)
```

As a teaser, here you can see the first 6 rows of this long table. Or you can download the CSV file to your computer from [GitHub](https://github.com/LieberInstitute/recount3-docs/blob/master/recount3_metadata_files.csv). If you want to, you can combine it with the project raw files table from the previous section.

```{r "metadata_files_table", echo = FALSE}
metadata_files <- read.csv("recount3_metadata_files.csv", row.names = NULL)
head(metadata_files)
```

The URL structure is:

`<recount3_url>`/`<organism>`/data_sources/`<data_source>`/metadata/`<last 2 project letters or digits>`/`<project>`/`<data_source>`.`<table name>`.`<project>`.MD.gz

## Sample-level BigWig files

Each sample in recount3 has bigWig file publicly available and whose URL can be obtained using `recount3::locate_url()`. Below we show the URL for one such sample. 

```{r "example_bigWig_URL"}
locate_url(
    "SRP009615",
    "data_sources/sra",
    type = "bw",
    sample = "SRR387777"
)
```

The URL structure is:

`<recount3_url>`/`<organism>`/data_sources/`<data_source>`/base_sums/`<last 2 project letters or digits>`/`<project>`/`<last 2 sample letters or digits>`/`<data_source>`.base_sums.`<project>`_`<sample>`.ALL.bw

Valid `recount3_url` options we support are `http://duffel.rail.bio/recount3` and `http://idies.jhu.edu/recount3/data`.
