<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>recount3: human and mouse public RNA-seq data</title>
  <meta name="description" content="recount3: human and mouse public RNA-seq data" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="recount3: human and mouse public RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="recount3: human and mouse public RNA-seq data" />
  
  
  

<meta name="author" content="" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="data.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a><ul>
<li><a href="index.html#related-recount3-websites">Related <code>recount3</code> websites</a></li>
<li><a href="index.html#how-to-cite-recount3">How to cite <code>recount3</code></a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#related-projects"><i class="fa fa-check"></i>Related projects</a></li>
<li><a href="index.html#previous-recount-projects">Previous <code>ReCount</code> projects</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#related-publications"><i class="fa fa-check"></i>Related publications</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#team-members"><i class="fa fa-check"></i>Team members</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#teams-involved"><i class="fa fa-check"></i>Teams involved</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
</ul></li>
<li class="part"><span><b>I Get RNA-seq data!</b></span></li>
<li class="chapter" data-level="1" data-path="data.html"><a href="data.html"><i class="fa fa-check"></i><b>1</b> Data</a><ul>
<li class="chapter" data-level="1.1" data-path="data.html"><a href="data.html#human"><i class="fa fa-check"></i><b>1.1</b> Human</a></li>
<li class="chapter" data-level="1.2" data-path="data.html"><a href="data.html#mouse"><i class="fa fa-check"></i><b>1.2</b> Mouse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="quick-access.html"><a href="quick-access.html"><i class="fa fa-check"></i><b>2</b> Quick Access</a><ul>
<li class="chapter" data-level="2.1" data-path="index.html"><a href="index.html#recount3"><i class="fa fa-check"></i><b>2.1</b> recount3</a></li>
<li class="chapter" data-level="2.2" data-path="quick-access.html"><a href="quick-access.html#snapcount"><i class="fa fa-check"></i><b>2.2</b> snapcount</a></li>
<li class="chapter" data-level="2.3" data-path="quick-access.html"><a href="quick-access.html#megadepth"><i class="fa fa-check"></i><b>2.3</b> megadepth</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="study-explorer.html"><a href="study-explorer.html"><i class="fa fa-check"></i><b>3</b> Study explorer</a></li>
<li class="chapter" data-level="4" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html"><i class="fa fa-check"></i><b>4</b> How to ask for help</a><ul>
<li class="chapter" data-level="4.1" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#gitter"><i class="fa fa-check"></i><b>4.1</b> Gitter</a></li>
<li class="chapter" data-level="4.2" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#bioconductor-support-website"><i class="fa fa-check"></i><b>4.2</b> Bioconductor Support website</a></li>
<li class="chapter" data-level="4.3" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#github-issues"><i class="fa fa-check"></i><b>4.3</b> GitHub Issues</a></li>
</ul></li>
<li class="part"><span><b>II Details</b></span></li>
<li class="chapter" data-level="5" data-path="quality-check-fields.html"><a href="quality-check-fields.html"><i class="fa fa-check"></i><b>5</b> Quality check fields</a></li>
<li class="chapter" data-level="6" data-path="bioconductor.html"><a href="bioconductor.html"><i class="fa fa-check"></i><b>6</b> Bioconductor</a><ul>
<li class="chapter" data-level="6.1" data-path="index.html"><a href="index.html#recount3"><i class="fa fa-check"></i><b>6.1</b> recount3</a></li>
<li class="chapter" data-level="6.2" data-path="quick-access.html"><a href="quick-access.html#snapcount"><i class="fa fa-check"></i><b>6.2</b> snapcount</a></li>
<li class="chapter" data-level="6.3" data-path="quick-access.html"><a href="quick-access.html#megadepth"><i class="fa fa-check"></i><b>6.3</b> megadepth</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="raw-files.html"><a href="raw-files.html"><i class="fa fa-check"></i><b>7</b> Raw files</a><ul>
<li class="chapter" data-level="7.1" data-path="raw-files.html"><a href="raw-files.html#data-source-vs-collection"><i class="fa fa-check"></i><b>7.1</b> Data source vs collection</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="bigwig-tools.html"><a href="bigwig-tools.html"><i class="fa fa-check"></i><b>8</b> bigWig tools</a></li>
<li class="chapter" data-level="9" data-path="related-software.html"><a href="related-software.html"><i class="fa fa-check"></i><b>9</b> Related software</a></li>
<li class="chapter" data-level="10" data-path="related-analyses.html"><a href="related-analyses.html"><i class="fa fa-check"></i><b>10</b> Related analyses</a></li>
<li class="part"><span><b>III Expand <code>ReCount</code></b></span></li>
<li class="chapter" data-level="11" data-path="your-data.html"><a href="your-data.html"><i class="fa fa-check"></i><b>11</b> Your data</a><ul>
<li class="chapter" data-level="11.1" data-path="your-data.html"><a href="your-data.html#process-your-data"><i class="fa fa-check"></i><b>11.1</b> Process your data</a></li>
<li class="chapter" data-level="11.2" data-path="your-data.html"><a href="your-data.html#contribute-your-collections"><i class="fa fa-check"></i><b>11.2</b> Contribute your collections</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="join-the-recount-team.html"><a href="join-the-recount-team.html"><i class="fa fa-check"></i><b>12</b> Join the <code>ReCount</code> team!</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">recount3: human and mouse public RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">recount3: human and mouse public RNA-seq data</h1>
</div>
<div id="overview" class="section level1 unnumbered">
<h1>Overview</h1>
<p>Welcome to the home of the <code>recount3</code> project! This website is the home to the most updated version of all the documentation related to the <code>recount3</code> project.</p>
<p><strong>recount3</strong> is an online resource consisting of RNA-seq gene, exon, and exon-exon junction counts as well as coverage bigWig files for 8,679 and 10,088 different studies for human and mouse respectively. It is the third generation of the <a href="http://bowtie-bio.sourceforge.net/recount/">ReCount project</a>.</p>
<p>The raw sequencing data were processed with <a href="https://github.com/langmead-lab/recount-pump/"><code>recount-pump</code></a> as described in the <a href="https://biorxiv.org">recount3 paper</a> which created the coverage bigWig files and the recount-unified text files. While these raw output files are available through <a href="">SciServer</a>, for ease of statistical analysis, we provide through the <a href="https://bioconductor.org/packages/recount3"><code>recount3</code></a> R/Bioconductor package an interface that builds <a href="https://bioconductor.org/packages/SummarizedExperiment">RangedSummarizedExperiment</a> R objects for gene, exon, and exon-exon junction counts. The coverage bigWig files can be used for annotation-agnostic expression analyses using for example <a href="https://bioconductor.org/packages/megadepth"><code>megadepth</code></a>, <a href="https://bioconductor.org/packages/derfinder"><code>derfinder</code></a> and other tools.</p>
<p>By taking care of several pre-processing steps and combining many datasets into one easily-accessible website, we make finding and analyzing RNA-seq data considerably more straightforward.</p>
<div id="related-recount3-websites" class="section level2 unnumbered">
<h2>Related <code>recount3</code> websites</h2>
<p><code>recount3</code> is composed of several related software tools and websites. Here is the full list, with all of them referring to this website for their unified documentation.</p>
<ul>
<li><a href="https://github.com/langmead-lab/recount-pump/"><code>recount-pump</code></a>: TODO</li>
<li><code>recount-unifier</code>: TODO</li>
<li><code>monorail</code>: TODO</li>
<li><code>megadepth</code>: software <a href="https://github.com/ChristopherWilks/megadepth">source</a>, R package <a href="https://github.com/LieberInstitute/megadepth">source</a>, Bioconductor <a href="https://bioconductor.org/packages/megadepth">landing page</a>, <code>pkgdown</code> documentation <a href="https://LieberInstitute.github.io/megadepth">website</a></li>
<li><code>recount3</code>: R package <a href="https://github.com/LieberInstitute/recount3">source</a>, Bioconductor <a href="https://bioconductor.org/packages/recount3">landing page</a>, <code>pkgdown</code> documentation <a href="https://LieberInstitute.github.io/recount3">website</a>.</li>
<li><a href="https://github.com/LieberInstitute/recount3-docs"><code>recount3-docs</code></a>: the source code for this documentation <a href="https://LieberInstitute.github.io/recount3-docs">website</a>.</li>
</ul>
</div>
<div id="how-to-cite-recount3" class="section level2 unnumbered">
<h2>How to cite <code>recount3</code></h2>
<p>Thank you for your continued support of the <code>ReCount</code> family of projects! We greatly appreciate you citing our work.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="index.html#cb1-1"></a><span class="kw">print</span>(<span class="kw">citation</span>(<span class="st">&quot;recount3&quot;</span>)[<span class="dv">2</span>], <span class="dt">bibtex =</span> <span class="ot">TRUE</span>)</span></code></pre></div>
<pre><code>## 
## Wilks C, Collado-Torres L, Zheng SC, Jaffe AE, Nellore A, Hansen KD,
## Langmead B (2020). &quot;recount3 pre-print title TODO.&quot; _bioRxiv_. doi:
## 10.1101/TODO (URL: https://doi.org/10.1101/TODO), &lt;URL:
## https://www.biorxiv.org/content/10.1101/TODO&gt;.
## 
## A BibTeX entry for LaTeX users is
## 
##   @Article{,
##     title = {recount3 pre-print title TODO},
##     author = {Christopher Wilks and Leonardo Collado-Torres and Shijie C. Zheng and Andrew E. Jaffe and Abhinav Nellore and Kasper D. Hansen and Ben Langmead},
##     year = {2020},
##     journal = {bioRxiv},
##     doi = {10.1101/TODO},
##     url = {https://www.biorxiv.org/content/10.1101/TODO},
##   }</code></pre>
<div id="related-projects" class="section level3 unnumbered">
<h3>Related projects</h3>
<p>The <code>ReCount</code> project has three main publications over the years with implementation details that have been improved each time. Each phase of the project has had several companion publications that expanded the main resource in different directions. Please cite the appropriate expansions and results as you see relevant for your work.</p>
</div>
<div id="previous-recount-projects" class="section level3 unnumbered">
<h3>Previous <code>ReCount</code> projects</h3>
<ul>
<li><a href="https://jhubiostatistics.shinyapps.io/recount/"><code>recount2</code></a></li>
<li><a href="http://bowtie-bio.sourceforge.net/recount/"><code>ReCount</code></a></li>
</ul>
</div>
<div id="related-publications" class="section level3 unnumbered">
<h3>Related publications</h3>
<div id="recount3" class="section level4 unnumbered">
<h4><code>recount3</code></h4>
</div>
<div id="recount2" class="section level4 unnumbered">
<h4><code>recount2</code></h4>
</div>
<div id="recount" class="section level4 unnumbered">
<h4><code>ReCount</code></h4>
</div>
</div>
</div>
<div id="team-members" class="section level2 unnumbered">
<h2>Team members</h2>
<ul>
<li><a href="https://github.com/">Christopher Wilks</a> (the star behind <code>recount3</code>!)</li>
<li><a href="https://github.com/lcolladotor">Leonardo Collado Torres</a></li>
<li><a href="https://github.com/">Shijie Charles Zheng</a></li>
</ul>
</div>
<div id="teams-involved" class="section level2 unnumbered">
<h2>Teams involved</h2>
<ul>
<li><a href="http://www.langmead-lab.org/">Ben Langmead’s lab at JHU Computer Science</a></li>
<li><a href="https://www.hansenlab.org/">Kasper Daniel Hansen’s lab at JHBSPH Biostatistics Department</a></li>
<li><a href="http://lcolladotor.github.io/">Leonardo Collado-Torres</a> and <a href="http://aejaffe.com/">Andrew E. Jaffe</a> from <a href="https://www.libd.org/">LIBD</a></li>
<li><a href="http://nellore.bio/">Abhinav Nellore’s lab at OHSU</a></li>
<li><a href="http://jtleek.com/">Jeff Leek’s lab at JHBSPH Biostatistics Deparment</a></li>
<li>Data hosted by <a href="https://www.sciserver.org/">SciServer from IDIES at JHU</a></li>
</ul>
<table>
<tbody>
<tr class="odd">
<td><a href="http://www.langmead-lab.org/"><img src="http://www.langmead-lab.org/wp-content/uploads/2014/01/Screen-Shot-2014-02-02-at-5.20.13-PM-1024x199.png" width="250px"></a></td>
<td><a href="https://www.libd.org/"><img src="http://lcolladotor.github.io/img/LIBD_logo.jpg" width="250px"></a></td>
<td><a href="http://nellore.bio/"><img src="https://seekvectorlogo.net/wp-content/uploads/2018/08/oregon-health-science-university-ohsu-vector-logo.png" width="250px"></a></td>
<td><a href="https://www.sciserver.org/"><img src="https://skyserver.sdss.org/dr14/en/images/sciserver_logo_inverted_vertical.png" width="250px"></a></td>
</tr>
</tbody>
</table>
<p>This book was last updated on 2020-10-23 21:16:04.</p>
</div>
<div id="r-session-information" class="section level2 unnumbered">
<h2>R session information</h2>
<p>Details on the R version used for making this book. The source code is available at <a href="https://github.com/lcolladotor/bioc_team_ds"><code>lcolladotor/bioc_team_ds</code></a>.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="index.html#cb3-1"></a><span class="co">## Load the package at the top of your script</span></span>
<span id="cb3-2"><a href="index.html#cb3-2"></a><span class="kw">library</span>(<span class="st">&quot;sessioninfo&quot;</span>)</span>
<span id="cb3-3"><a href="index.html#cb3-3"></a></span>
<span id="cb3-4"><a href="index.html#cb3-4"></a><span class="co">## Reproducibility information</span></span>
<span id="cb3-5"><a href="index.html#cb3-5"></a><span class="kw">print</span>(<span class="st">&#39;Reproducibility information:&#39;</span>)</span>
<span id="cb3-6"><a href="index.html#cb3-6"></a><span class="kw">Sys.time</span>()</span>
<span id="cb3-7"><a href="index.html#cb3-7"></a><span class="kw">proc.time</span>()</span>
<span id="cb3-8"><a href="index.html#cb3-8"></a><span class="kw">options</span>(<span class="dt">width =</span> <span class="dv">120</span>)</span>
<span id="cb3-9"><a href="index.html#cb3-9"></a><span class="kw">session_info</span>()</span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="index.html#cb4-1"></a><span class="co">## For links</span></span>
<span id="cb4-2"><a href="index.html#cb4-2"></a><span class="kw">library</span>(<span class="st">&quot;BiocStyle&quot;</span>)</span>
<span id="cb4-3"><a href="index.html#cb4-3"></a></span>
<span id="cb4-4"><a href="index.html#cb4-4"></a><span class="co">## Packages illustrated in the documentation</span></span>
<span id="cb4-5"><a href="index.html#cb4-5"></a><span class="kw">library</span>(<span class="st">&quot;recount3&quot;</span>)</span>
<span id="cb4-6"><a href="index.html#cb4-6"></a><span class="kw">library</span>(<span class="st">&quot;megadepth&quot;</span>)</span></code></pre></div>
<pre><code>## [1] &quot;Reproducibility information:&quot;</code></pre>
<pre><code>## [1] &quot;2020-10-23 21:16:13 UTC&quot;</code></pre>
<pre><code>##    user  system elapsed 
##   8.835   0.406   9.277</code></pre>
<pre><code>## ─ Session info ───────────────────────────────────────────────────────────────────────────────────────────────────────
##  setting  value                                      
##  version  R version 4.0.3 Patched (2020-10-21 r79360)
##  os       Ubuntu 20.04.1 LTS                         
##  system   x86_64, linux-gnu                          
##  ui       X11                                        
##  language (EN)                                       
##  collate  en_US.UTF-8                                
##  ctype    en_US.UTF-8                                
##  tz       Etc/UTC                                    
##  date     2020-10-23                                 
## 
## ─ Packages ───────────────────────────────────────────────────────────────────────────────────────────────────────────
##  package              * version  date       lib source                                    
##  assertthat             0.2.1    2019-03-21 [2] RSPM (R 4.0.0)                            
##  Biobase              * 2.49.1   2020-09-03 [1] Bioconductor                              
##  BiocFileCache          1.13.1   2020-08-04 [1] Bioconductor                              
##  BiocGenerics         * 0.35.4   2020-06-04 [1] Bioconductor                              
##  BiocManager          * 1.30.10  2019-11-16 [2] CRAN (R 4.0.3)                            
##  BiocParallel           1.23.3   2020-10-17 [1] Bioconductor                              
##  BiocStyle            * 2.17.1   2020-09-24 [1] Bioconductor                              
##  Biostrings             2.57.2   2020-06-09 [1] Bioconductor                              
##  bit                    4.0.4    2020-08-04 [1] RSPM (R 4.0.2)                            
##  bit64                  4.0.5    2020-08-30 [1] RSPM (R 4.0.2)                            
##  bitops                 1.0-6    2013-08-17 [1] RSPM (R 4.0.0)                            
##  blob                   1.2.1    2020-01-20 [1] RSPM (R 4.0.0)                            
##  bookdown               0.20     2020-06-23 [1] RSPM (R 4.0.2)                            
##  cli                    2.1.0    2020-10-12 [2] RSPM (R 4.0.2)                            
##  cmdfun                 1.0.2    2020-10-10 [1] RSPM (R 4.0.2)                            
##  crayon                 1.3.4    2017-09-16 [2] RSPM (R 4.0.0)                            
##  curl                   4.3      2019-12-02 [2] RSPM (R 4.0.0)                            
##  data.table             1.13.0   2020-07-24 [1] RSPM (R 4.0.2)                            
##  DBI                    1.1.0    2019-12-15 [1] RSPM (R 4.0.0)                            
##  dbplyr                 1.4.4    2020-05-27 [1] RSPM (R 4.0.0)                            
##  DelayedArray           0.15.16  2020-10-06 [1] Bioconductor                              
##  digest                 0.6.25   2020-02-23 [2] RSPM (R 4.0.0)                            
##  dplyr                  1.0.2    2020-08-18 [1] RSPM (R 4.0.2)                            
##  ellipsis               0.3.1    2020-05-15 [2] RSPM (R 4.0.0)                            
##  evaluate               0.14     2019-05-28 [2] RSPM (R 4.0.0)                            
##  fansi                  0.4.1    2020-01-08 [2] RSPM (R 4.0.0)                            
##  fs                     1.5.0    2020-07-31 [2] RSPM (R 4.0.2)                            
##  generics               0.0.2    2018-11-29 [1] RSPM (R 4.0.0)                            
##  GenomeInfoDb         * 1.25.11  2020-09-03 [1] Bioconductor                              
##  GenomeInfoDbData       1.2.4    2020-10-23 [1] Bioconductor                              
##  GenomicAlignments      1.25.3   2020-06-09 [1] Bioconductor                              
##  GenomicRanges        * 1.41.6   2020-08-12 [1] Bioconductor                              
##  glue                   1.4.2    2020-08-27 [2] RSPM (R 4.0.2)                            
##  hms                    0.5.3    2020-01-08 [1] RSPM (R 4.0.0)                            
##  htmltools              0.5.0    2020-06-16 [2] RSPM (R 4.0.1)                            
##  httr                   1.4.2    2020-07-20 [2] RSPM (R 4.0.2)                            
##  IRanges              * 2.23.10  2020-06-13 [1] Bioconductor                              
##  knitr                  1.30     2020-09-22 [2] RSPM (R 4.0.2)                            
##  lattice                0.20-41  2020-04-02 [3] CRAN (R 4.0.3)                            
##  lifecycle              0.2.0    2020-03-06 [2] RSPM (R 4.0.0)                            
##  magrittr               1.5      2014-11-22 [2] RSPM (R 4.0.0)                            
##  Matrix                 1.2-18   2019-11-27 [3] CRAN (R 4.0.3)                            
##  MatrixGenerics       * 1.1.8    2020-10-20 [1] Bioconductor                              
##  matrixStats          * 0.57.0   2020-09-25 [1] RSPM (R 4.0.2)                            
##  megadepth            * 0.99.10  2020-10-23 [1] Github (LieberInstitute/megadepth@358abe9)
##  memoise                1.1.0    2017-04-21 [2] RSPM (R 4.0.0)                            
##  pillar                 1.4.6    2020-07-10 [2] RSPM (R 4.0.2)                            
##  pkgconfig              2.0.3    2019-09-22 [2] RSPM (R 4.0.0)                            
##  purrr                  0.3.4    2020-04-17 [2] RSPM (R 4.0.0)                            
##  R.methodsS3            1.8.1    2020-08-26 [1] RSPM (R 4.0.2)                            
##  R.oo                   1.24.0   2020-08-26 [1] RSPM (R 4.0.2)                            
##  R.utils                2.10.1   2020-08-26 [1] RSPM (R 4.0.2)                            
##  R6                     2.4.1    2019-11-12 [2] RSPM (R 4.0.0)                            
##  rappdirs               0.3.1    2016-03-28 [1] RSPM (R 4.0.0)                            
##  Rcpp                   1.0.5    2020-07-06 [2] RSPM (R 4.0.2)                            
##  RCurl                  1.98-1.2 2020-04-18 [1] RSPM (R 4.0.0)                            
##  readr                  1.4.0    2020-10-05 [1] RSPM (R 4.0.2)                            
##  recount3             * 0.99.11  2020-10-23 [1] Github (LieberInstitute/recount3@ce25093) 
##  rlang                  0.4.8    2020-10-08 [2] RSPM (R 4.0.2)                            
##  rmarkdown              2.4      2020-09-30 [1] RSPM (R 4.0.2)                            
##  Rsamtools              2.5.3    2020-06-26 [1] Bioconductor                              
##  RSQLite                2.2.1    2020-09-30 [1] RSPM (R 4.0.2)                            
##  rstudioapi             0.11     2020-02-07 [2] RSPM (R 4.0.0)                            
##  rtracklayer            1.49.5   2020-08-07 [1] Bioconductor                              
##  S4Vectors            * 0.27.14  2020-10-09 [1] Bioconductor                              
##  sessioninfo          * 1.1.1    2018-11-05 [1] RSPM (R 4.0.0)                            
##  stringi                1.5.3    2020-09-09 [2] RSPM (R 4.0.2)                            
##  stringr                1.4.0    2019-02-10 [2] RSPM (R 4.0.0)                            
##  SummarizedExperiment * 1.19.9   2020-10-01 [1] Bioconductor                              
##  tibble                 3.0.4    2020-10-12 [2] RSPM (R 4.0.2)                            
##  tidyselect             1.1.0    2020-05-11 [1] RSPM (R 4.0.0)                            
##  vctrs                  0.3.4    2020-08-29 [2] RSPM (R 4.0.2)                            
##  withr                  2.3.0    2020-09-22 [2] RSPM (R 4.0.2)                            
##  xfun                   0.18     2020-09-29 [2] RSPM (R 4.0.2)                            
##  XML                    3.99-0.5 2020-07-23 [1] RSPM (R 4.0.2)                            
##  XVector                0.29.3   2020-06-25 [1] Bioconductor                              
##  yaml                   2.2.1    2020-02-01 [2] RSPM (R 4.0.0)                            
##  zlibbioc               1.35.0   2020-04-27 [1] Bioconductor                              
## 
## [1] /usr/local/lib/R/host-site-library
## [2] /usr/local/lib/R/site-library
## [3] /usr/local/lib/R/library</code></pre>

</div>
</div>



            </section>

          </div>
        </div>
      </div>

<a href="data.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
