<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 Quick Access | recount3: human and mouse public RNA-seq data</title>
  <meta name="description" content="2 Quick Access | recount3: human and mouse public RNA-seq data" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="2 Quick Access | recount3: human and mouse public RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 Quick Access | recount3: human and mouse public RNA-seq data" />
  
  
  

<meta name="author" content="" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="data.html"/>
<link rel="next" href="study-explorer.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Overview</a><ul>
<li><a href="index.html#related-recount3-websites">Related <code>recount3</code> websites</a></li>
<li><a href="index.html#how-to-cite-recount3">How to cite <code>recount3</code></a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#related-projects"><i class="fa fa-check"></i>Related projects</a></li>
<li><a href="index.html#previous-recount-projects">Previous <code>ReCount</code> projects</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#related-publications"><i class="fa fa-check"></i>Related publications</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#team-members"><i class="fa fa-check"></i>Team members</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#teams-involved"><i class="fa fa-check"></i>Teams involved</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#r-session-information"><i class="fa fa-check"></i>R session information</a></li>
</ul></li>
<li class="part"><span><b>I Get RNA-seq data!</b></span></li>
<li class="chapter" data-level="1" data-path="data.html"><a href="data.html"><i class="fa fa-check"></i><b>1</b> Data</a><ul>
<li class="chapter" data-level="1.1" data-path="data.html"><a href="data.html#human"><i class="fa fa-check"></i><b>1.1</b> Human</a></li>
<li class="chapter" data-level="1.2" data-path="data.html"><a href="data.html#mouse"><i class="fa fa-check"></i><b>1.2</b> Mouse</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="quick-access.html"><a href="quick-access.html"><i class="fa fa-check"></i><b>2</b> Quick Access</a><ul>
<li class="chapter" data-level="2.1" data-path="index.html"><a href="index.html#recount3"><i class="fa fa-check"></i><b>2.1</b> recount3</a></li>
<li class="chapter" data-level="2.2" data-path="quick-access.html"><a href="quick-access.html#snapcount"><i class="fa fa-check"></i><b>2.2</b> snapcount</a></li>
<li class="chapter" data-level="2.3" data-path="quick-access.html"><a href="quick-access.html#megadepth"><i class="fa fa-check"></i><b>2.3</b> megadepth</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="study-explorer.html"><a href="study-explorer.html"><i class="fa fa-check"></i><b>3</b> Study explorer</a></li>
<li class="chapter" data-level="4" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html"><i class="fa fa-check"></i><b>4</b> How to ask for help</a><ul>
<li class="chapter" data-level="4.1" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#gitter"><i class="fa fa-check"></i><b>4.1</b> Gitter</a></li>
<li class="chapter" data-level="4.2" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#bioconductor-support-website"><i class="fa fa-check"></i><b>4.2</b> Bioconductor Support website</a></li>
<li class="chapter" data-level="4.3" data-path="how-to-ask-for-help.html"><a href="how-to-ask-for-help.html#github-issues"><i class="fa fa-check"></i><b>4.3</b> GitHub Issues</a></li>
</ul></li>
<li class="part"><span><b>II Details</b></span></li>
<li class="chapter" data-level="5" data-path="quality-check-fields.html"><a href="quality-check-fields.html"><i class="fa fa-check"></i><b>5</b> Quality check fields</a></li>
<li class="chapter" data-level="6" data-path="bioconductor.html"><a href="bioconductor.html"><i class="fa fa-check"></i><b>6</b> Bioconductor</a><ul>
<li class="chapter" data-level="6.1" data-path="index.html"><a href="index.html#recount3"><i class="fa fa-check"></i><b>6.1</b> recount3</a></li>
<li class="chapter" data-level="6.2" data-path="quick-access.html"><a href="quick-access.html#snapcount"><i class="fa fa-check"></i><b>6.2</b> snapcount</a></li>
<li class="chapter" data-level="6.3" data-path="quick-access.html"><a href="quick-access.html#megadepth"><i class="fa fa-check"></i><b>6.3</b> megadepth</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="raw-files.html"><a href="raw-files.html"><i class="fa fa-check"></i><b>7</b> Raw files</a><ul>
<li class="chapter" data-level="7.1" data-path="raw-files.html"><a href="raw-files.html#data-source-vs-collection"><i class="fa fa-check"></i><b>7.1</b> Data source vs collection</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="bigwig-tools.html"><a href="bigwig-tools.html"><i class="fa fa-check"></i><b>8</b> bigWig tools</a></li>
<li class="chapter" data-level="9" data-path="related-software.html"><a href="related-software.html"><i class="fa fa-check"></i><b>9</b> Related software</a></li>
<li class="chapter" data-level="10" data-path="related-analyses.html"><a href="related-analyses.html"><i class="fa fa-check"></i><b>10</b> Related analyses</a></li>
<li class="part"><span><b>III Expand <code>ReCount</code></b></span></li>
<li class="chapter" data-level="11" data-path="your-data.html"><a href="your-data.html"><i class="fa fa-check"></i><b>11</b> Your data</a><ul>
<li class="chapter" data-level="11.1" data-path="your-data.html"><a href="your-data.html#process-your-data"><i class="fa fa-check"></i><b>11.1</b> Process your data</a></li>
<li class="chapter" data-level="11.2" data-path="your-data.html"><a href="your-data.html#contribute-your-collections"><i class="fa fa-check"></i><b>11.2</b> Contribute your collections</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="join-the-recount-team.html"><a href="join-the-recount-team.html"><i class="fa fa-check"></i><b>12</b> Join the <code>ReCount</code> team!</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">recount3: human and mouse public RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="quick-access" class="section level1">
<h1><span class="header-section-number">2</span> Quick Access</h1>
<div id="recount3" class="section level2">
<h2><span class="header-section-number">2.1</span> recount3</h2>
<p><em>The following information is taken from the <a href="https://bioconductor.org/packages/recount3"><code>recount3</code></a> documentation.</em></p>
<p>After installing <em><a href="https://bioconductor.org/packages/3.12/recount3">recount3</a></em>, we need to load the package, which will automatically load the required dependencies.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="quick-access.html#cb9-1"></a><span class="co">## Load recount3 R package</span></span>
<span id="cb9-2"><a href="quick-access.html#cb9-2"></a><span class="kw">library</span>(<span class="st">&quot;recount3&quot;</span>)</span></code></pre></div>
<p>If you have identified a <strong>study</strong> of interest and want to access the gene level expression data, use <code>create_rse()</code> as shown below. <code>create_rse()</code> has arguments that will allow you to specify the <strong>annotation</strong> of interest for the given organism, and whether you want to download <strong>gene</strong>, <strong>exon</strong> or <strong>exon-exon junction</strong> expression data.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="quick-access.html#cb10-1"></a><span class="co">## Find all available human projects</span></span>
<span id="cb10-2"><a href="quick-access.html#cb10-2"></a>human_projects &lt;-<span class="st"> </span><span class="kw">available_projects</span>()</span>
<span id="cb10-3"><a href="quick-access.html#cb10-3"></a></span>
<span id="cb10-4"><a href="quick-access.html#cb10-4"></a><span class="co">## Find the project you are interested in,</span></span>
<span id="cb10-5"><a href="quick-access.html#cb10-5"></a><span class="co">## here we use SRP009615 as an example</span></span>
<span id="cb10-6"><a href="quick-access.html#cb10-6"></a>proj_info &lt;-<span class="st"> </span><span class="kw">subset</span>(</span>
<span id="cb10-7"><a href="quick-access.html#cb10-7"></a>    human_projects,</span>
<span id="cb10-8"><a href="quick-access.html#cb10-8"></a>    project <span class="op">==</span><span class="st"> &quot;SRP009615&quot;</span> <span class="op">&amp;</span><span class="st"> </span>project_type <span class="op">==</span><span class="st"> &quot;data_sources&quot;</span></span>
<span id="cb10-9"><a href="quick-access.html#cb10-9"></a>)</span>
<span id="cb10-10"><a href="quick-access.html#cb10-10"></a></span>
<span id="cb10-11"><a href="quick-access.html#cb10-11"></a><span class="co">## Create a RangedSummarizedExperiment (RSE) object at the gene level</span></span>
<span id="cb10-12"><a href="quick-access.html#cb10-12"></a>rse_gene_SRP009615 &lt;-<span class="st"> </span><span class="kw">create_rse</span>(proj_info)</span>
<span id="cb10-13"><a href="quick-access.html#cb10-13"></a></span>
<span id="cb10-14"><a href="quick-access.html#cb10-14"></a><span class="co">## Explore that RSE object</span></span>
<span id="cb10-15"><a href="quick-access.html#cb10-15"></a>rse_gene_SRP009615</span></code></pre></div>
<pre><code>## class: RangedSummarizedExperiment 
## dim: 63856 12 
## metadata(8): time_created recount3_version ... annotation recount3_url
## assays(1): raw_counts
## rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ... ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
## rowData names(10): source type ... havana_gene tag
## colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
## colData names(175): rail_id external_id ... recount_pred.curated.cell_line BigWigURL</code></pre>
<p>You can also interactively choose your study of interest</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="quick-access.html#cb12-1"></a><span class="co">## Note that you can interactively explore the available projects</span></span>
<span id="cb12-2"><a href="quick-access.html#cb12-2"></a>proj_info_interactive &lt;-<span class="st"> </span>interactiveDisplayBase<span class="op">::</span><span class="kw">display</span>(human_projects)</span>
<span id="cb12-3"><a href="quick-access.html#cb12-3"></a></span>
<span id="cb12-4"><a href="quick-access.html#cb12-4"></a><span class="co">## Select a single row, then hit &quot;send&quot;. The following code checks this.</span></span>
<span id="cb12-5"><a href="quick-access.html#cb12-5"></a><span class="kw">stopifnot</span>(<span class="kw">nrow</span>(proj_info_interactive) <span class="op">==</span><span class="st"> </span><span class="dv">1</span>)</span>
<span id="cb12-6"><a href="quick-access.html#cb12-6"></a></span>
<span id="cb12-7"><a href="quick-access.html#cb12-7"></a><span class="co">## Then create the RSE object</span></span>
<span id="cb12-8"><a href="quick-access.html#cb12-8"></a>rse_gene_interactive &lt;-<span class="st"> </span><span class="kw">create_rse</span>(proj_info_interactive)</span></code></pre></div>
<p>Once you have a RSE file, you can use <code>transform_counts()</code> to transform the raw coverage counts.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="quick-access.html#cb13-1"></a><span class="co">## Once you have your RSE object, you can transform the raw coverage</span></span>
<span id="cb13-2"><a href="quick-access.html#cb13-2"></a><span class="co">## base-pair coverage counts using transform_counts().</span></span>
<span id="cb13-3"><a href="quick-access.html#cb13-3"></a><span class="co">## For RPKM, TPM or read outputs, check the details in transform_counts().</span></span>
<span id="cb13-4"><a href="quick-access.html#cb13-4"></a><span class="kw">assay</span>(rse_gene_SRP009615, <span class="st">&quot;counts&quot;</span>) &lt;-<span class="st"> </span><span class="kw">transform_counts</span>(rse_gene_SRP009615)</span></code></pre></div>
<p>Now you are ready to continue with downstream analysis software.</p>
<p><em><a href="https://bioconductor.org/packages/3.12/recount3">recount3</a></em> also supports accessing the BigWig raw coverage files as well as specific study or collection sample <strong>metadata</strong>. Please continue to the <a href="https://bioconductor.org/packages/recount3">users guide</a> for more detailed information.</p>
</div>
<div id="snapcount" class="section level2">
<h2><span class="header-section-number">2.2</span> snapcount</h2>
</div>
<div id="megadepth" class="section level2">
<h2><span class="header-section-number">2.3</span> megadepth</h2>
<p><em>The following information is taken from the <a href="https://bioconductor.org/packages/megadepth"><code>megadepth</code></a> documentation.</em></p>
<p>To get started, we need to load the <em><a href="https://bioconductor.org/packages/3.12/megadepth">megadepth</a></em> package into our R session. This will load all the required dependencies.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="quick-access.html#cb14-1"></a><span class="kw">library</span>(<span class="st">&quot;megadepth&quot;</span>)</span></code></pre></div>
<p>Once we have the R package loaded, we need to install the <a href="https://github.com/ChristopherWilks/megadepth">Megadepth</a> software. We can do so with <code>install_megadepth()</code>, which downloads a binary for your OS (Linux, Windows or macOS).<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> We can then use with an example BigWig file to compute the coverage at a set of regions.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="quick-access.html#cb15-1"></a><span class="co">## Install the latest version of Megadepth</span></span>
<span id="cb15-2"><a href="quick-access.html#cb15-2"></a><span class="kw">install_megadepth</span>(<span class="dt">force =</span> <span class="ot">TRUE</span>)</span></code></pre></div>
<pre><code>## The latest megadepth version is 1.0.7</code></pre>
<pre><code>## megadepth has been installed to /github/home/bin</code></pre>
<p>Next, we might want to use <em><a href="https://bioconductor.org/packages/3.12/megadepth">megadepth</a></em> to quantify the coverage at a set of regions of the genome of interest to us. Here we will use two example files that are include in the package for illustration and testing purposes. One of them is a <a href="https://genome.ucsc.edu/goldenPath/help/bigWig.html">bigWig file</a> that contains the base-pair coverage information for a sample of interest and the second one is <a href="https://genome.ucsc.edu/FAQ/FAQformat.html#format1">BED file</a> which contains the genomic region coordinates of interest. So we first locate them.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="quick-access.html#cb18-1"></a><span class="co">## Next, we locate the example BigWig and annotation files</span></span>
<span id="cb18-2"><a href="quick-access.html#cb18-2"></a>example_bw &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">&quot;tests&quot;</span>, <span class="st">&quot;test.bam.all.bw&quot;</span>,</span>
<span id="cb18-3"><a href="quick-access.html#cb18-3"></a>    <span class="dt">package =</span> <span class="st">&quot;megadepth&quot;</span>, <span class="dt">mustWork =</span> <span class="ot">TRUE</span></span>
<span id="cb18-4"><a href="quick-access.html#cb18-4"></a>)</span>
<span id="cb18-5"><a href="quick-access.html#cb18-5"></a>annotation_file &lt;-<span class="st"> </span><span class="kw">system.file</span>(<span class="st">&quot;tests&quot;</span>, <span class="st">&quot;testbw2.bed&quot;</span>,</span>
<span id="cb18-6"><a href="quick-access.html#cb18-6"></a>    <span class="dt">package =</span> <span class="st">&quot;megadepth&quot;</span>, <span class="dt">mustWork =</span> <span class="ot">TRUE</span></span>
<span id="cb18-7"><a href="quick-access.html#cb18-7"></a>)</span>
<span id="cb18-8"><a href="quick-access.html#cb18-8"></a></span>
<span id="cb18-9"><a href="quick-access.html#cb18-9"></a><span class="co">## Where are they?</span></span>
<span id="cb18-10"><a href="quick-access.html#cb18-10"></a>example_bw</span></code></pre></div>
<pre><code>## [1] &quot;/usr/local/lib/R/host-site-library/megadepth/tests/test.bam.all.bw&quot;</code></pre>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="quick-access.html#cb20-1"></a>annotation_file</span></code></pre></div>
<pre><code>## [1] &quot;/usr/local/lib/R/host-site-library/megadepth/tests/testbw2.bed&quot;</code></pre>
<p>Once we have located the example files we can proceed to calculating the base-pair coverage for our genomic regions of interest. There are several ways to do this with <em><a href="https://bioconductor.org/packages/3.12/megadepth">megadepth</a></em>, but here we use the user-friendly function <code>get_coverage()</code>. This function will perform a given operation <strong>op</strong> on the bigWig file for a given set of regions of interest (<em>annotation</em>). One of those operations is to compute the mean base-pair coverage for each input region. This is what we’ll do with our example bigWig file.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="quick-access.html#cb22-1"></a><span class="co">## We can then use megadepth to compute the coverage</span></span>
<span id="cb22-2"><a href="quick-access.html#cb22-2"></a>bw_cov &lt;-<span class="st"> </span><span class="kw">get_coverage</span>(</span>
<span id="cb22-3"><a href="quick-access.html#cb22-3"></a>    example_bw,</span>
<span id="cb22-4"><a href="quick-access.html#cb22-4"></a>    <span class="dt">op =</span> <span class="st">&quot;mean&quot;</span>,</span>
<span id="cb22-5"><a href="quick-access.html#cb22-5"></a>    <span class="dt">annotation =</span> annotation_file</span>
<span id="cb22-6"><a href="quick-access.html#cb22-6"></a>)</span>
<span id="cb22-7"><a href="quick-access.html#cb22-7"></a>bw_cov</span></code></pre></div>
<pre><code>## GRanges object with 4 ranges and 1 metadata column:
##         seqnames          ranges strand |     score
##            &lt;Rle&gt;       &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt;
##   [1]      chr10            0-10      * |      0.00
##   [2]      chr10 8756697-8756762      * |     15.85
##   [3]      chr10 4359156-4359188      * |      3.00
##   [4] GL000219.1   168500-168620      * |      1.26
##   -------
##   seqinfo: 2 sequences from an unspecified genome; no seqlengths</code></pre>
<p><code>get_coverage()</code> returns an object that is familiar to <em><a href="https://bioconductor.org/packages/3.12/GenomicRanges">GenomicRanges</a></em> users, that is, a <code>GRanges</code> object that can be used with other Bioconductor software packages.</p>
<p>This example is just the tip of the iceberg, as <a href="https://github.com/ChristopherWilks/megadepth">Megadepth</a> and thus <em><a href="https://bioconductor.org/packages/3.12/megadepth">megadepth</a></em> can do a lot of useful processing operations on BAM and bigWig files.</p>
<p>Please continue to the <a href="https://bioconductor.org/packages/megadepth">users guide</a> for more detailed information.</p>

</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>Please check <a href="https://github.com/ChristopherWilks/megadepth">Megadepth</a> for instructions on how to compile the software from source if the binary version doesn’t work for you.<a href="quick-access.html#fnref1" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="study-explorer.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
